remote-monitoring-tests.sln

```diff
@@ -25,6 +26,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "scripts", "scripts", "{1968
 		scripts\storageadapter.sh = scripts\storageadapter.sh
 		scripts\telemetry-agent.sh = scripts\telemetry-agent.sh
 		scripts\telemetry.sh = scripts\telemetry.sh
+		scripts\containers.sh = scripts\containers.sh
```

Comment:
I don't see this file in the PR and the name seems incorrect

Reply:
This file was already added to the solution but not in the .sln file. I added it so it could be seen in visual studio. I can submit a separate PR if needed for the one line change.

Reply:
I see. I had a look at the script and apparently what it does it starts all the containers. The script doesn't have comments and the name doesn't help either, could you mention it to @ppathan pls?

Reply:
The purpose is to stop/start all containers. It should have comments though :)

Reply:
You should edit that file to start ASA manager too, right?. Please add this at the end:

```
error "No command specified, pass either 'start' or 'stop'."
exit 1
```

Reply:
done